packages/portal/app/components/buttons/SubmitButton.tsx

```diff
@@ -1,4 +1,6 @@
-import { Button, forwardRef, type ButtonProps } from "@chakra-ui/react";
+import { forwardRef } from "@chakra-ui/react";
+
+import { Button, ButtonProps } from "@components/buttons";
```

Comment:
Since this is within the `components/buttons` folder can you use a relative import?

```suggestion
import { Button, ButtonProps } from "./Button";
```